src/shared/sam/cli/samCliLocalInvoke.ts

```diff
@@ -106,6 +110,15 @@ export class DefaultSamLocalInvokeCommand implements SamLocalInvokeCommand {
                     } else if (code !== 0) {
                         reject(new Error(`"${samCommandName}" command stopped (error code: ${code})`))
                     }
+
+                    // Forces debugger to disconnect (sometimes it fails to disconnect on its own)
+                    // Note that VSCode 1.42 only allows us to get the active debug session, so
+                    // the user will have to manually disconnect if using multiple debug sessions
```

Comment:
we could pass around the `debugSession` object if we wanted to support multiple debug sessions. But that is a backlog/TODO.